docs/docs/features/soft-off.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Soft Off Feature
+sidebar_label: Soft Off
+---
+
+Similar to the deep sleep feature that sends the keyboard into a low power state after a certain period of inactivity, the soft off feature is used to turn the keyboard on and off explicitly. Depending on the keyboard, this may be through a dedicated on/off push button, or merely through an additional binding in the keymap to turn the device off and the existing reset button to turn the device back on.
+
+The feature is intended as an alternative to using a hardware switch to physically cut power from the battery to the keyboard. This can be useful for existing PCBs not designed for wireless that don't have a power switch, or for new designs that favor a push button on/off like found on other devices.
+
+:::note
+
+The power off is accomplished by putting the MCU into a "soft off" state. Power is _not_ technically removed from the entire system, but the device will only be woken from the state by a few possible events.
+
+:::
+
+Once powered off, the keyboard will only wake up when:
+
+- You press the same button/sequence that you pressed to power off the keyboard, or
+- You press a reset button found on the keyboard.
+
+## Soft Off With Existing Designs
+
+For existing designs, using soft off is as simple as placing the [Soft Off Behavior](../behaviors/soft-off.md) in your keymap and then invoking it. For splits, at least for now, you'll need to place it somewhere on each side of your keymap and trigger on both sides, starting from the peripheral side first.
+
+You can then wake up the keyboard by pressing the reset button once, and repeating this for each side for split keyboards.
+
+## Adding Soft On/Off To New Designs
```

Comment:
```suggestion
## Adding A Dedicated Soft On/Off Button To New Designs
```
I think updating this header would clarify that this is talking about the first approach mentioned earlier in:
> Depending on the keyboard, this may be through a dedicated on/off push button, or merely through an additional binding in the keymap to turn the device off and the existing reset button to turn the device back on.


This is technically not an individual button in case of hardware combo, but do we have a better name for this "dedicated" approach?

Reply:
Perhaps we should focus on "Dedicated On/Off GPIO Pin" as a concept?

Reply:
Yeah, that sounds clear and precise.

Reply:
Updated the header.